modes/basic-test-mode/src/toolbarButtons.ts

```diff
@@ -248,6 +251,19 @@ const toolbarButtons: Button[] = [
       evaluate: 'evaluate.cornerstoneTool',
     },
   },
+  {
```

Comment:
We need to figure out a way for the extensions to define default buttons too, so that it isn't necesary for every mode to know about all the button definitions unless they are creating new, custom ones.